sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/SchedulerConfig.java

```diff
@@ -179,6 +180,11 @@
      */
     private static final String LIBPROCESS_IP_ENVVAR = "LIBPROCESS_IP";
 
+    /**
+     * Environment variable to disable zk data migration from mono service schema to multi service schema.
+     */
+    private static final String DISABLE_MONO_TO_MULTI_MIGRATION = "DISABLE_MONO_TO_MULTI_MIGRATION";
```

Comment:
I think this can be removed. It's only referenced within hello-world. In practice I think any additional gating on migration should be developer-owned anyway.